ValeStyles/Couchbase/See.yml

```diff
@@ -2,11 +2,7 @@ extends: existence
 message: Use "see" for links.
 level: warning
 link: https://developers.google.com/style/link-text
-
+scope: raw
 nonword: true
 tokens: 
-  - '[Rr]efer to http://'
-  - '[Rr]efer to https://'
-  - '[Rr]efer to xref:.+.adoc\[.+\]'
-  - '[Rr]efer to {.+}\[.+\]'
-  - '[Rr]efer to <<.+>>'
\ No newline at end of file
+  - '[Rr]efer to\s+(xref:[^\s]+|https:\/\/[^\s]+|http:\/\/[^\s]+|<<[^>]+>>)'
```

Comment:
I snuck this one in, too, because the original wasn't working. 

Scope needs to be `raw` or it'll ignore the http/https links.